README.md

```diff
@@ -2,30 +2,86 @@ Core ML Community Tools
 =======================
 
 Core ML community tools contains all supporting tools for Core ML model
-conversion and validation. This includes scikit-learn, LIBSVM, Caffe,
-Keras and XGBoost.
+conversion, editing and validation. This includes scikit-learn, LIBSVM, Caffe,
+Keras, Tensorflow and XGBoost.
```

Comment:
Consider reordering and put the more commonly used ones (e.g. TensorFlow) in front?

Reply:
"This includes deep learning frameworks like Tensorflow, Keras, Caffe as well as classical machine learning frameworks like LIBSVB, scikit-learn, and XGBoost.

Reply:
Done.